fv/vxlan_test.go

```diff
@@ -133,6 +134,10 @@ var _ = infrastructure.DatastoreDescribe("_BPF-SAFE_ VXLAN topology before addin
 					}
 					for _, felix := range felixes {
 						felix.Stop()
+
+						// see if blackhole routes go away with felix
+						o, _ := felix.ExecOutput("ip", "r", "s", "type", "blackhole")
```

Comment:
After calling Stop, any Exec will fail because the container is down so this test isn't useful.  Suggest you just remove it.